### PR TITLE
Time all test in travis.

### PR DESCRIPTION
**Pull request by nouiz**
I think we should always keep that. I think it don't really change the tests time and will help to speed up the tests to always have it.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ script:
   - cd -; cd Theano
   - python -c 'import theano; print(theano.config.__str__(print_doc=False))'
   - python -c 'import theano; assert(theano.config.blas.ldflags != "")'
-  - theano-nose -v $PART
+  - theano-nose -v --with-timer --timer-top-n 10 $PART
```
Comment (abergeron) [inline .travis.yml]: Why not keep all the timings?

**Comment by abergeron** [comment]
I'm ok with always using nose-timer.  I'm not so sure about the cProfile or theano profiling part.  Mostly because they add output and I really dislike chatty tests.

**Comment by nouiz** [comment]
I'll remove the theano profiler and cprofiler I added. They are in this
branch now to help identify the problem.

Having a too long output isn't great. I'm ok to changing the size from 10
to something else, but I don't like having it too big like 100.

Do you think 20 would be better then 10? 10 have been enough to help me
debug the slowness. We can increase it later if needed.

**Comment by nouiz** [comment]
issue https://github.com/Theano/Theano/issues/4507 is about droping python 3.3

**Pull request closed without merging**